### PR TITLE
Clean ticket buyer fields

### PR DESCRIPTION
**Pull request by vctt94**
This is a pull request for solving the issue https://github.com/decred/decrediton/issues/594

Also this pull request is refactoring the inputs for creating a general DecredInput and NumericInput

### CODE REVIEW

File: app/components/DecredInput/DcrInput.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        requiredMessage= {requiredMessage}
+        disabled={disabled}
+        readOnly={readOnly}
+        placeholder={placeholder}
+        required= {required}
+        invalid= {invalid}
+        value={value}
+        onChange={onChange}
+      />
+      <Balance
```
Comment (go1dfish) [inline app/components/DecredInput/DcrInput.js]: I don’t think we need to show the balance again but we should show the unit type and eventually a drop down for atoms vs DCR or other future units.

File: app/components/DecredInput/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import DcrInput from "./DcrInput";
+import { autobind } from "core-decorators";
+import balanceConnector from "../../connectors/balance";
+
+@autobind
+class DecredInput extends React.Component {
+  constructor(props) {
+    super(props);
```
Comment (go1dfish) [inline app/components/DecredInput/index.js]: No need for this constructor if you are not going to do anything with it

File: app/components/DecredInput/DcrInput.js (hunk 1 above)
Comment (vctt94) [reply app/components/DecredInput/DcrInput.js]: What drop down would be that?

**Comment by alexlyp** [comment]
default Ticket fee, Tx fee overflow the inputs in the expanded ticket purchase area

**Comment by alexlyp** [comment]
when an error is given on ticket fee and tx fee i see none, but it seems to push the Expiry field to the next line

**Comment by alexlyp** [comment]
AutoBuyer information button needs padding to match the Ticket Purchase information button above

**Comment by alexlyp** [comment]
Empty fields in Auto buyer area need to throw errors as well.  Currently I'm able to turn on the autobuyer with empty fields

**Comment by alexlyp** [comment]
You should be able to have a balance to maintain higher than your balance

**Comment by alexlyp** [comment]
Ticket Fee and Tx Fee should also have sensible max values (0.1 DCR is probably good there)

**Comment by alexlyp** [comment]
Max Fee field needs to respond to DCR --> atoms change as well.

**Comment by alexlyp** [comment]
Errors for fields in the Auto Buyer config should be below the label and the field.  Currently they are just under the field and cause bad shifting when shown

**Comment by alexlyp** [comment]
Ok looks good except for one issue.  When you first go to the purchase tickets form and enter an invalid value into ticket fee or tx fee, you don't get any errors displayed until AFTER clicking purchase.  Once purchase is clicked it works as expected.  Fix that this PR should be good to go.

File: app/components/views/TicketsPage/Page.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import StakeyBounce from "../../StakeyBounce";
```
Comment (alexlyp) [inline app/components/views/TicketsPage/Page.js]: This entire Page.js can be removed.

**Pull request merged**